<ui version="4.0">
  <author></author>
  <comment></comment>
  <exportmacro></exportmacro>
  <class>QFitParWidget</class>
  <widget class="QWidget">
    <property name="geometry" >
      <rect>
        <x>0</x>
        <y>0</y>
        <width>304</width>
        <height>339</height>
      </rect>
    </property>
    <property name="windowTitle" >
      <string>Parameter</string>
    </property>
    <widget class="QCheckBox" name="FixedChk" >
      <property name="geometry" >
        <rect>
          <x>20</x>
          <y>180</y>
          <width>57</width>
          <height>23</height>
        </rect>
      </property>
      <property name="text" >
        <string>fixed</string>
      </property>
    </widget>
    <widget class="QGo4LineEdit" name="ValueEdt" >
      <property name="geometry" >
        <rect>
          <x>60</x>
          <y>100</y>
          <width>142</width>
          <height>27</height>
        </rect>
      </property>
    </widget>
    <widget class="QCheckBox" name="RangeChk" >
      <property name="geometry" >
        <rect>
          <x>20</x>
          <y>220</y>
          <width>64</width>
          <height>23</height>
        </rect>
      </property>
      <property name="text" >
        <string>range</string>
      </property>
    </widget>
    <widget class="QLabel" name="TextLabel2" >
      <property name="geometry" >
        <rect>
          <x>10</x>
          <y>140</y>
          <width>40</width>
          <height>18</height>
        </rect>
      </property>
      <property name="text" >
        <string>Error</string>
      </property>
      <property name="wordWrap" >
        <bool>false</bool>
      </property>
    </widget>
    <widget class="QGo4LineEdit" name="EpsilonEdt" >
      <property name="geometry" >
        <rect>
          <x>100</x>
          <y>290</y>
          <width>142</width>
          <height>27</height>
        </rect>
      </property>
    </widget>
    <widget class="QGo4LineEdit" name="ErrorEdt" >
      <property name="geometry" >
        <rect>
          <x>60</x>
          <y>140</y>
          <width>142</width>
          <height>27</height>
        </rect>
      </property>
    </widget>
    <widget class="QCheckBox" name="EpsilonChk" >
      <property name="geometry" >
        <rect>
          <x>20</x>
          <y>290</y>
          <width>73</width>
          <height>23</height>
        </rect>
      </property>
      <property name="text" >
        <string>epsilon</string>
      </property>
    </widget>
    <widget class="QGo4LineEdit" name="RangeMaxEdt" >
      <property name="geometry" >
        <rect>
          <x>100</x>
          <y>240</y>
          <width>142</width>
          <height>27</height>
        </rect>
      </property>
    </widget>
    <widget class="QLabel" name="TextLabel1" >
      <property name="geometry" >
        <rect>
          <x>10</x>
          <y>100</y>
          <width>46</width>
          <height>18</height>
        </rect>
      </property>
      <property name="text" >
        <string>Value</string>
      </property>
      <property name="wordWrap" >
        <bool>false</bool>
      </property>
    </widget>
    <widget class="QGo4LineEdit" name="RangeMinEdt" >
      <property name="geometry" >
        <rect>
          <x>100</x>
          <y>200</y>
          <width>142</width>
          <height>27</height>
        </rect>
      </property>
    </widget>
  </widget>
  <layoutdefault spacing="6" margin="11" />
  <customwidgets>
    <customwidget>
      <class>QGo4LineEdit</class>
      <extends>QLineEdit</extends>
      <header>QGo4LineEdit.h</header>
      <container>0</container>
    </customwidget>
  </customwidgets>
</ui>
